tools/walletextension/storage/storage.go

```diff
@@ -11,10 +11,14 @@ import (
 	"github.com/ten-protocol/go-ten/tools/walletextension/storage/database/sqlite"
 )
 
+// todo - pass the Context
```

Comment:
is this todo for this PR or should we do it in the next one?

Reply:
for a future pr.